docs/api/config/js_kanban_cardshape_config.md

```diff
@@ -266,13 +279,17 @@ const cardShape = { // card settings
     },
     users: {
         show: true,
-        values: users
+        values: users,
+        maxCount: 3
```

Comment:
4 or something, mb `false` since users sometimes want to remove the limit altogether